### DISCUSSION TITLE
Doorbell on/off - 2. relay function

### DISCUSSION THREAD

**Discussion by geraldFS** [omersiar/esp-rfid#590] [Ideas]
Hello,

since the project can also signal the doorbell, it would be great if you could switch a relay on and off with which you can mute the doorbell.

So a relay with a simple on-off logic.
If possible with an icon of a doorbell

If this would work, almost all of my wishes would be fulfilled.

Best regards
Gerald

**Comment by matjack1** [comment]
Hey @geraldFS wouldn't be a better place a relay that you control manually that disconnects the doorbell?

**Reply by geraldFS** [reply]
Well, I'd like to turn off the bell when I get off night duty. Since I manage everything centrally with Homeassistant, I would like to save myself another device.

And in my hardware I have already installed two relays and also one pin for detection when someone presses the bell.

The only thing missing would be the software implementation.

**Reply by matjack1** [reply]
What about implementing that in home assistant instead? I feel like manually enabling/disabling the bell from esp-rfid is error prone. You need to remember to click every day. Otherwise you need a scheduler, but this is too big of a feature for the esp-rfid, I think.

While adding a cronjob that enables/disables the doorbell in HA seems like a better place to implement this?

I'm trying to resist to pack more features in esp-rfid since we are already pushing the ESP8266 to the limits and we should shrink the application if possible to make it more stable :)

**Comment by geraldFS** [comment]
well no more functions but an improvement of the existing ones.

Simply keep the second relay ON - and it is only deactivated by clicking on it or via an MQTT command.

However, it would only be necessary to specify the type that is sent to Homeassistant.

Status (button/switch)
ON/OFF or closed/open
and probably the icon which is then created.

Currently I have done exactly the same, but I always have to manually create the switch in Homeassistant.

**Reply by matjack1** [reply]
I think that adding a control for an additional relay for this functionality is too much. We are already short for pins and this functionality can be managed pretty easily in a third-party system, like you do with Home Assistant

What do you mean that you have to manually create the switch in HA? Isn't that something you do once?
